src/heartratetask/HeartRateTask.cpp

```diff
@@ -85,21 +101,23 @@ void HeartRateTask::Work() {
 }
 
 void HeartRateTask::PushMessage(HeartRateTask::Messages msg) {
-  BaseType_t xHigherPriorityTaskWoken = pdFALSE;
+  BaseType_t xHigherPriorityTaskWoken;
+  xHigherPriorityTaskWoken = pdFALSE;
```

Comment:
I don't think this change was intentional.

Reply:
@FintasticMan This was present when I started this PR. @JF002 was this done during the WaspOS port? What should be done with this code?

Reply:
I was talking about the change from `BaseType_t xHigherPriorityTaskWoken = pdFALSE;` on one line to these 2 lines.

Reply:
@FintasticMan Got it. Not sure why I changed it. Most likely a clang-tidy misunderstanding on my part.